project/Build.scala

```diff
@@ -1316,14 +1336,30 @@ object Build {
     ).
     settings(
       Compile / resourceGenerators += Def.task {
-        val jsDestinationFile = (Compile / resourceManaged).value / "dotty_res" / "scripts" / "scaladoc-scalajs.js"
-        sbt.IO.copyFile((`scaladoc-js` / Compile / fullOptJS).value.data, jsDestinationFile)
-        Seq(jsDestinationFile)
+        val contributorsFile = (`scaladoc-js-contributors` / Compile / fullOptJS).value.data
+        val contributorsDestinationFile = Paths.get("docs-for-dotty-page", "js", "contributors.js").toFile
+        sbt.IO.copyFile(contributorsFile, contributorsDestinationFile)
+
+        val mainFile = (`scaladoc-js-main` / Compile / fullOptJS).value.data
+        val mainDestinationFile = (Compile / resourceManaged).value / "dotty_res" / "scripts" / "scaladoc-scalajs.js"
+        sbt.IO.copyFile(mainFile, mainDestinationFile)
+
+        Seq(mainDestinationFile, contributorsDestinationFile)
       }.taskValue,
       Compile / resourceGenerators += Def.task {
-        Seq("code-snippets.css", "searchbar.css", "social-links.css", "ux.css", "versions-dropdown.css").map { file =>
```

Comment:
What do you think of moving all the “logic” to a separate file (in `project/DocumentationWebsite.scala`, for instance) and having just one line here:

~~~ scala
Compile / resourceGenerators += Def.task {
  DocumentationWebsite.generateStaticAssets(
    (`scaladoc-js-contributors` / Compile / fullOptJS).value,
    (`scaladoc-js-main` / Compile / fullOptJS).value,
    (`scaladoc-js-common` / Compile / resourceDirectory).value,
    ...
    (Compile / resourceManaged).value,
  )
}.taskValue
~~~

Reply:
I can do that